src/qz/ui/LinkLabel.java

```diff
@@ -34,22 +34,21 @@ public LinkLabel() {
     public LinkLabel(final String text) {
         super(linkify(text));
         initialize();
-        addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                try {
-                    // Sense the action based on the content of the text
-                    if (text.contains("@")) {
-                        Desktop.getDesktop().mail(new URI(text));
-                    } else {
-                        File filePath = new File(text);
-                        ShellUtilities.browseDirectory(filePath.isDirectory() ? text : filePath.getParent());
-                    }
-
-                }
-                catch(Exception ex) {
-                    log.error("", ex);
+        addActionListener(e -> {
+            try {
+                // Sense the action based on the content of the text
+                if (text.contains("@")) {
+                    //TODO This will cause gtk errors on elementary... I think...
+                    Desktop.getDesktop().mail(new URI(text));
```

Comment:
I don't even know what to do here. Desktop.getDesktop() will immediately cause an uncatchable error if Gtk 3 is running. Is there a generic way to do this on linux?